shared/container.go

```diff
@@ -34,7 +34,7 @@ func IsUint32(value string) error {
 		return nil
 	}
 
-	_, err := strconv.ParseInt(value, 10, 32)
+	_, err := strconv.ParseUint(value, 10, 32)
```

Comment:
Just checking because you changed this before: `uint32` is the correct type here and not `uint64`?